### PR TITLE
Remove Generic dialect

### LINKED ISSUE #451: Remove `Generic` implementation as it's been migrated to is own repository
Remove `Generic` implementation as it's been migrated to is own repository

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Closes #451.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,28 +142,9 @@ Additional resources:
 
 ### Run Time Dependencies
```
Comment (AnastasiiaSergienko) [inline README.md:143]: I think We should remove all dependencies from here and from the pom,  but leave the pom be till some next pull request

File: README.md [hunk 2]
```diff
@@ -78,6 +79,30 @@ in all your `CREATE JAVA ADAPTER SCRIPT` statements. The old entry point was rem
 * Allows redirecting log output to a remote machine
 * Allows remote debugging with the Java Debugger
 
+## Supported Data Sources
```
Comment (AnastasiiaSergienko) [inline README.md:82]: This looks like a duplication of the section above.  We should have only one of them left

Comment (AnastasiiaSergienko) [reply README.md:82]: Maybe this table is more readable

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (chiaradiamarcelo) [reply README.md:82]: yes, totally. I just put it to have your feedback.

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**